packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -285,6 +285,13 @@ library LibProving {
 
         if (!skipZKPVerification) {
             for (uint256 i = 0; i < config.zkProofsPerBlock; ++i) {
+                bytes32 instance = keccak256(
```

Comment:
We should decide that `blockHash`, `prover` adn `txListHash` are enough to verify the proof.
As previous version, we need more information, like: history hashes, previous state root etc.

Reply:
Maybe these fields are not necessary as @Brechtpd said

Reply:
These are witness, we do not need them.